### ISSUE TITLE
Too much unnecessary information announced by default

### ISSUE THREAD

**Issue by Gene703122** [nvaccess/nvda#13407]
### Is your feature request related to a problem? Please describe.

A lot of unnecessary information is announced in browse mode and perhaps elsewhere.  On web pages, I constantly hear "figure."  Unless I'm designing or working on a web page, this is useless clutter.  All sorts of announcements of shapes like right ;pointing triangle, left pointing triangle, and myriad others are announced.  Announcement of emphasis is extremely distracting.  No human reader would do so unless the person  being read to requested it.  

### Describe the solution you'd like

this sort of material should be off by default.  users advanced enough will learn about turning announcements on they want.  There should be a single command to turn a good deal of announcements on or off for convenience so the user doesn't have to change multiple settings.  Of course, the user should be able to turn on and off individual announcements.  the multiple command is for the convenience of users.

### Describe alternatives you've considered

I don't see any.  

### Additional context

the event that finally got me to write a feature request is a message on an NVDA list by someone complaining about hearing "decorative text" in a Word document every time such an image appears, and this document has a great many of them.  Of course, the user should be able to have such information announced but this is another example of what is for a significant majority of users, unnecessary clutter.  The user says this cannot be turned off.  

Much more care should be given to what is announced by default.  NVDA is increasingly announcing information that is distracting, time wasting, and of no value to the majority of users who aren't designing or working on web pages and perhaps elsewhere.  These sorts of announcements are relevant for such work, not for the typical blind reader.  Its as though a sighted reader announced italics and other attributes every time they are encountered in a book.  No one expects this unless the person being read to needs this information for a specific purpose.

**Comment by Qchristensen** [comment]
(I edited the issue text as there were strings of spaces before each line which made GitHub present it as blockquoted text and made it harder to read).

I'm also going to close this as it is too vague and the two specific points mentioned are completely unrelated to each other.  Please create one issue for each point.  EG create one issue describing the problem  and context with "figure", and a separate issue for decorate images

**Comment by CyrilleB79** [comment]
Hi @Gene703122 

First of all and to help understand can you clarify the following points:
* are you a blind user, a person working or helping for blind people (e.g. working in accessibility field) or in another situation?
* did you open this ticket for yourself or for someone else?

If I summarize your request, you do not want that any information other than text to be reported by NVDA. That's not always a good idea and I will try to explain why.
Please also keep in mind when writing bug reports or feature requests that what seem obvious for you is not necessarily for others. Hence the request of many details on what does not work and what you expect instead.

You write:
> A lot of unnecessary information is announced in browse mode and perhaps elsewhere. On web pages, I constantly hear "figure."

Personally, I do not hear this frequently while navigating on the web. Probably it depends on the webpages you and I are reading which are different. Hence the request to give precise examples and links to the webpage where the issue occurs.

You write:
> All sorts of announcements of shapes like right ;pointing triangle, left pointing triangle, and myriad others are announced. 

I agree that hearing reports like "right ;pointing triangle" may be distracting. Since I do not have the webpage example, I cannot be sure. But I think that this report is due to special characters. You can disable them by unchecking the following option in "Speech settings": "Include Unicode Consortium data (including emoji) in character and symbol processing".
But if you do, you will also disable the reporting of emojis such as smiling face, thumb up. And some of them convey some meaning. Do you really want to ignore them?
Also right-pointing triangle character is sometimes used in lists in replacement of bullets; in this case ignoring these characters can lead to a wrong perception of the text structure, i.e. you do not realize anymore that you are in a list.

> Announcement of emphasis is extremely distracting.

Regarding my personal experience, emphasis is rarely used, at least on the webpage that I read. If I recall correctly, NVAccess devs have chosen to keep it enabled because it conveys more meaning than just formatting such as bold or italic.
But again, having examples of webpage where it is distracting should help.

> No human reader would do so unless the person being read to requested it.

IMO, that's not a justification.
For example, no human reader would either read a webpage content indicating "link" for each link. But personally, when reading a webpage, I need to have such information to know where I can click and get some more resource.
The only pages where I do not want to have links reported are Wikipedia or dictionary pages; but that's the exception.

> Describe the solution you'd like
> this sort of material should be off by default. users advanced enough will learn about turning announcements on they want.

If you think to NVDA options that should be off by default, please provide the list. Try to go to Document formatting settings, uncheck all the checkboxes and browse the web. You will see that you miss a lot of information about how the text is structured (e.g. titles, links, etc.)

> There should be a single command to turn a good deal of announcements on or off for convenience so the user doesn't have to change multiple settings. Of course, the user should be able to turn on and off individual announcements. the multiple command is for the convenience of users.

It is already possible if you set up a profile where you disable all the options that create to much "noise". You can then assign a shortcut to enable or disable manually this profile.
If you request this profile to be created by default in NVDA, you should indicate clearly which options should be disabled in this profile, I mean, something more precise than "a good deal of announcements". As explained above, which announcement should be disabled is not so obvious.

> Additional context
> the event that finally got me to write a feature request is a message on an NVDA list by someone complaining about hearing "decorative text" in a Word document every time such an image appears, and this document has a great many of them.

To be clear, the person who asked to disable such announcement did not express on other announcements such as emoji, emphasis, etc.

> Much more care should be given to what is announced by default. NVDA is increasingly announcing information that is distracting, time wasting, and of no value to the majority of users who aren't designing or working on web pages and perhaps elsewhere. These sorts of announcements are relevant for such work, not for the typical blind reader.

Be careful not to generalize. Some people may complain that NVDA is too verbose. But if we do not have precise example of what is announced and should not, we cannot establish that all these persons are disturbed by the same type of announcements. Hence the request to have a precise list of options or announcements to be disabled and associated examples.

> Its as though a sighted reader announced italics and other attributes every time they are encountered in a book. No one expects this unless the person being read to needs this information for a specific purpose.

Actually formatting information such as italic or bold is not reported by default. But keep in mind that a printed text also conveys information just by the way it is formatted. E.g. a sighted person is able to see the title and the start of a new paragraph; a blind person should have the same level of information to be able to understand the content of a document.


At last, I would like to indicate that as a ticket author, you can reopen the ticket and explain why if you feel that it should not be closed.
For example, maybe you can edit this ticket to concentrate on a set of options that should be disabled by default if it's your main request. Let's forget the issue of decorative figures that is not the same;

That's it. I have taken the time to answer to all what you write so that you can understand why your request has ben considered vague and not obvious.

Hope this helps.

**Comment by Sylvie42** [comment]
Regarding the specific problem of decorative images pronounced by NVDA in office applications, see [issue 13422](https://github.com/nvaccess/nvda/issues/13422).

**Comment by Gene703122** [comment]
My response is long so read it, if you wish to, at your convenience.  
    
    I am a user.  I created this issue mostly not for me.  I know enough about NVDA to stop the behaviors, but there are a lot of people who don't.  
    
    Regarding the assertion that the issues are unrelated, why?  If you are saying they are unrelated as a developer because the solutions are different, from a user standpoint, my standpoint, that is not relevant.  the issues are related.  In both cases, material is spoken and there is either no way or no reasonable way in settings to turn them off.  
    
    To answer your question, shapes such as right pointing triangle, left pointing triangle, filled right pointing triangle etc. are what I am referring to.  There is a way to stop the announcement of shapes such as right pointing triangles, filled left pointing triangles and perhaps fifty or more other shapes.  These are spoken anywhere they occur, but I usually see them on web pages.  I can manually set every shape in Punctuation/symbol pronunciation to all so they are not announced in lower punctuation levels but that is hardly a reasonable solution.  It was explained how to stop the announcement by turning off one setting and it was also said that that would stop announcements of all sorts of other information such as smiling face.  It is not my desire to stop all these other announcements and I never said it is.  I stated exactly what I wanted addressed.  My comments were not vague.  I am not being disrespectful, but you repeatedly broaden my comments in ways I did not intend nor in any way imply.  My goal is not that nothing but text be spoken and I neither said nor implied that.  Since I described specifically what I want changed, broadening my comments into areas I didn't mention is not a reasonable nor proper interpretation of what I said nor meant.  Since I discussed specific things and did not broaden nor imply a broader discussion, there is no basis to believe that I am making any such broad general statement.  I didn't say anything about links or headings or anything else.  
    
    Specifically, shape announcements should not be spoken by default.  Make them all be set to all punctuation setting level as a means of doing this if you like.  That would mean that they wouldn't be heard in general because you would only hear them if you are using the all punctuation level.  You would also hear all the other things such as smiling face, which I do not object to. Perhaps most would be the setting you might want to use.  What the setting should be is opened to discussion but it shouldn't be none and I don't think it should be some either.  And if that is the solution adopted, one command should be available to change all these settings.
    
    Also as I recall, the current setting is contradictory.  You say that right pointing triangles serve as bullets.  But as I recall, bullet is not announced in the none setting.  If one structure that serves the purpose of a bullet is announced, then the other should be announced under identical conditions.
    
    For those who care about the structure they are in, fine, they can have right pointing triangle set to be announced.  For typical reading, which is what most people do most of the time, what difference does it make if they know if they are still in a list or not.  If I'm reading an article and it lists the known bad effects of some chemical, I don't care if I am in a list.  I want to know what the effects are.  If I'm doing research, that may be different and if I want to have such information as right pointing triangle spoken before every item, than I can turn it on.  But this is not the appropriate default setting.  Something like all punctuation setting is.
    
    Regarding emphasis, while it may be rare, I seldom run into it, that still doesn't mean it should be announced by default.  And my discussion of what a reader reads is misinterpreted by you.  I have no objection to such things as links and headings being announced.  But a reader reading a book wouldn't announce such things because there are no such things.  I didn't talk about a sighted reader reading a web page and it wasn't my intent to discuss that.  A sighted reader reading a book doesn't announce every italic when it is in the text.  The reader determines what is important from context.  That is the context I am discussing and that I am applying here.  Information that conveys nothing important, that the listener derives from context, and that is distracting and serves no purpose for the general listener should be off by default.  A web page designer or editor may wish to turn such settings on.  
    
    Also, I didn't raise this issue just to raise these specific points.  I didn't address this in the initial discussion but the comments I am responding to caused me to think about it more and decide to raise it.  The philosophy of NVDA development seems to be that if something can be announced, it should be and will be by default.  While this isn't literally true for everything, it is strongly true as a generalization.  More and more information is announced as NVDA continues to develop, from shape after shape to figure.  
    
    How are these decisions made?  I think a survey should be done to see what people think of all this.  A lot of people just accept what a piece of technology like a screen-reader does.  Many people have an attitude toward computer and related technology that it does what it does.  They don't complain, they just accept it.  And a lot of people don't know how to change various default behaviors they may not like.  I submit that unless you do some sort of information gathering such as a survey, you have no basis to know whether your defaults such as I am describing are wanted by or useful to most users.
    
    A lot of your default settings are sound.  Some aren't.  I'm discussing two or three that aren't in my strong opinion.  
    
    Finally, I object to closing the topic.  In closing it almost immediately, comments by others are discouraged.  Perhaps others would show, in their comments, that closing the topic is an incorrect decision.  
    
    If I divided this issue into two separate ones, such as one for figure and the other for shapes, I would repeat the same general comments for each and the issues would be divided and diluted.

**Comment by CyrilleB79** [comment]
Hi @Gene703122

You have raised some interesting points even if I do not agree with all of them. I will not answer to all your last comment, else this issue will become really to wide and general.

Let's be more pragmatic. If you want to progress on this issue, I think that:
1. You should give an example of webpage representative of the issue, explain what you hear (e.g. by copy pasting the speech viewer content) and explain what you would like to hear instead.
2. And you should reopen this issue explaining that you have clarified it.

**Comment by seanbudd** [comment]
@Gene703122 @CyrilleB79 - A [GitHub Discussion](https://github.com/nvaccess/nvda/discussions/new) might serve as a better place for broader conversations on this topic. After forming specific issues there that are actionable, please open them as new issues.

**Comment by Gene703122** [comment]
At some point, I'll run across an example.  But it shouldn't be necessary to reproduce text from a web page.  

Here are examples of shapes that are announced followed by a fictitious newspaper paragraph where you might find such a shape repeatedly used.

As I said, these shapes may be used in all sorts of places.  I've found them on web pages.  

What I want is that they not be spoken by default.  
▴
▵
◃
↶
◐

The dangers of this chemical are principally four.
▷ It can deplete the oxygen level in a body of water, killing most of the fish population.
▷ It can corrode purification equipment.
▷ It can encourage the growth of algae in lesser concentrations.
▷ It can leak into soil, contaminating ground water.

In terms of reopening the issue, what does that mean?  Can I reopen this issue or do I have to open a completely new one?

**Comment by seanbudd** [comment]
@Gene703122  - for the example above, reopening this issue would be fine. If you come up with a fundamentally different issue, approach or solution, consider a new issue.
Thanks.

**Comment by Gene703122** [comment]
It was requested that I provide web sites where shapes are spoken.  Here is one.
On this page, if you search for back to top, you will hear what I reproduced below.

http://www.flying-blind.com/
Back to Top ⇧

How to those who determine what NVDA shall speak and I assume show in Braille by default, justify such shapes being announced?

**Comment by CyrilleB79** [comment]
@Gene703122, thanks for the example.

I agree that in the specific case that you exhibit, reporting the "⇧" is not necessary and pollutes the read text with a useless information. 
This is due to the following reason. The designer of this website has defined a link that triggers an action (go to top). The link contains the text describing the action ("Back to top"); but the link contains also a shape character ("⇧") used by an image that illustrates this action. If the webmaster had used an arrow image file with no alt description, screenreaders would not have heard anything. Unfortunately this is not the case.

On the contrary I come with another example where this character shape is used:
https://developer.huawei.com/consumer/en/doc/development/Tools-Guides/ide-shortcut-keys-0000001101336740
You can find a shortcut key written as follow:
"⇧+⌘+N"
The "⇧" character represents the shift key. If the screen reader ignores it, it is not possible to understand what is the shortcut key. This is because the webmaster of this site has used this character to describe the shift key (since there is such an arrow on this key). The character fully describes the key and unfortunately, the webmaster did not use an alternative text to describe that key. A text description of the key would have been better for screenreader users but graphics are probably nicer for sighted people. An other way to write it could have been to use an image of the key with an "alt" attribute describing it for screenreaders, that is "shift".

In conclusion, this just proves that there is no obvious way to report these characters since as shown by the two exhibited examples, their use is varied. Thus there should be a compromise between missing some information and avoid to be cluttered by useless information.
